crone/src/BufDiskWorker.cpp

```diff
@@ -340,7 +319,7 @@ noexcept {
     size_t bufFrames = buf.frames;
 
     size_t frSrc = secToFrame(startSrc);
-    clamp(frSrc, bufFrames - 1);
+    clamp(frSrc, file.frames() - 1);
```

Comment:
I think this is what is actually meant here: the starting point in the source file should be at least one frame before the end of the source file.